### PR TITLE
NDS-257 Brand refresh updates

### PR DESCRIPTION
**Pull request by barkertron**
https://nicedigital.atlassian.net/browse/NDS-257

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ These are the following breaking changes from 0.x to 1.x:
 - removed nunjucks templates, we now recommend using React components
 - removed the inverse panel
 - prefixed all mixin, functions, variables and classes with _nds-_ e.g. `container` is now `nds-container`
-- moved component-specific colour variables out of core into their respective component SCSS. Also renamed appropriately e.g.`$colour-panel-default` and `$colour-panel-default-border` are now in @nice-digital/nds-panel and renamed to `$nds-colour-panel-default-background` and `$nds-colour-panel-default-border`
+- moved component-specific colour variables out of core into their respective component SCSS. Also renamed appropriately e.g.`$colour-panel-default` and `$colour-panel-default-border` are now in @nice-digital/nds-panel and renamed to `$colour-panel-default-background` and `$colour-panel-default-border`
```
Comment (johndavey72) [inline README.md:141]: renamed colour-panel-default-border to colour-panel-default-border?

File: components/nds-action-banner/README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Any additional className will be merged on to the container.
 If you're not using [React](#react), then import the SCSS directly into your application by:
```
Comment (johndavey72) [inline components/nds-action-banner/README.md:99]: should this still say "import" ?

File: components/nds-alert/README.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ The styling to apply to the alert box. Defaults to `info` if not supplied. Can b
 If you're not using [React](#react), then import the SCSS directly into your application by:
```
Comment (johndavey72) [inline components/nds-alert/README.md:59]: should this still say import?

File: README.md (hunk 1 above)
Comment (barkertron) [reply README.md:141]: Don't worry about the docs - they're getting updated separately in the coming weeks as part of the design system review process

File: components/nds-tag/nds-tag.d.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ declare module "@nice-digital/nds-tag" {
 		/** Use the .tag--outline modifier for a tag with a border. Use outline tags with filtered lists to show the currently applied filters. */
 		outline?: boolean;
 		/** Use removeable tags when users need to be able to remove or dismiss a tag. Use with outline tags at the top of filtered lists to show the currently applied selections. */
```
Comment (johndavey72) [inline components/nds-tag/nds-tag.d.ts:15]: remove/removeable?

File: packages/docs/gatsby-config.js [hunk 5]
```diff
@@ -1,3 +1,10 @@
+const path = require("path");
+
+console.log(
```
Comment (johndavey72) [inline packages/docs/gatsby-config.js:3]: this ########### necessary?

File: components/nds-tag/nds-tag.d.ts (hunk 4 above)
Comment (barkertron) [reply components/nds-tag/nds-tag.d.ts:15]: Fair but a minor point that we probably don't want to delay the release for

File: packages/docs/gatsby-config.js (hunk 5 above)
Comment (barkertron) [reply packages/docs/gatsby-config.js:3]: Fair but a minor point that we probably don't want to delay the release for

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**